app/src/main/res/values/strings.xml

```diff
@@ -291,108 +291,109 @@
-    <string name="status_id_conversion_failed">Status id conversion failed.</string>
+    <string name="status_id_conversion_failed">Status ID conversion failed.</string>
     <string name="already_favourited">Already favourited.</string>
     <string name="already_boosted">Already boosted.</string>
     <string name="dont_screen_off">Keep the screen on when using the app.</string>
     <string name="hide_media_default">Hide all media</string>
-    <string name="cant_close_column_by_back_button_when_multiple_column_shown">You can\'t close the column using the back button when multiple columns are shown.</string>
+    <string name="cant_close_column_by_back_button_when_multiple_column_shown">You can\'t close the column using the \"←\" button with multiple columns visible.</string>
     <string name="disable_tablet_mode">Disable tablet mode (app restart required)</string>
-    <string name="minimum_column_width">Minimum column width (default=300(dp),app restart required)</string>
-    <string name="media_thumbnail_height">Media Thumbnail height (default=64(dp), app restart required)</string>
-    <string name="account_picker_reply">Which account do you reply from?</string>
-    <string name="reply_from_another_account">Reply from another account</string>
-    <string name="launcher_icon_by">Thanks to フタバ for the application icon.</string>
-    <string name="ssl_bug_7_0">Android 7.0 can only use the elliptic curve \"prime256v1\".\nUnfortunately your instance seems to not support it.\nThis bug has been fixed in Android 7.1.1.\nYou can either upgrade your OS or ask the administrator of your instance to add support to the elliptic curve \"prime256v1\".\nSee also https://code.google.com/p/android/issues/detail?id=224438</string>
+    <!-- Density-independent pixels, written as dp (pronounced “dips”), are flexible units that scale to have uniform dimensions on any screen. https://material.io/design/layout/understanding-layout.html#pixel-density -->
+    <string name="minimum_column_width">Minimum column width (default=300 dp), (app restart required)</string>
+    <string name="media_thumbnail_height">Media thumbnail height (default=64 dp), (app restart required)</string>
+    <string name="account_picker_reply">Reply from?</string>
+    <string name="reply_from_another_account">Reply from different account</string>
+    <string name="launcher_icon_by">Thanks to フタバ for the app icon.</string>
+    <string name="ssl_bug_7_0">Android 7.0 can only use the elliptic curve \"prime256v1\", which your instance doesn\t support.\nUpgrade to Android 7.1.1 or later, or add support for elliptic curve \"prime256v1\" on your instance.\nSee https://code.google.com/p/android/issues/detail?id=224438</string>
     <string name="timeline_font">Timeline font (app restart required)</string>
-    <string name="timeline_font_bold">Timeline font (bold, app restart required)</string>
+    <string name="timeline_font_bold">Bold (app restart required)</string>

```

Comment:
please don't remove "Timeline font"

Reply:
Possibly "Timeline font bold", is the convention, but it sounds a bit strict.